app/Http/Routes/Users.php

```diff
@@ -0,0 +1,8 @@
+<?php
```

Comment:
The routes and controllers should be in the same pull request. It's difficult to review without controller. please remove routes in this PR or commit your controller. Thanks